### PR TITLE
Data models

### LINKED ISSUE #19: Data model classes
- [x] create classes for current data model
- [x] add mock data

### PR DESCRIPTION
**Pull request by Ogochi**
resolves #19

### CODE REVIEW

**Comment by mbalc** [comment]
did you come to some crucial conclusions on our yesterday's talk?

File: server/src/main/java/com/omigost/server/data/model/Group.java [hunk 1]
```diff
@@ -0,0 +1,12 @@
+package com.omigost.server.data.model;
+
+import lombok.Data;
+
+import java.util.List;
+
+@Data
+public class Group {
+    long id;
```
Comment (mbalc) [inline server/src/main/java/com/omigost/server/data/model/Group.java]: We'll also have `List<Account> accounts` here I think

File: server/src/main/java/com/omigost/server/data/model/Account.java [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+import lombok.Data;
+
+import java.util.List;
+
+@Data
+public class Account {
+    long id;
+    String name;
+    List<Group> groups;
+    List<AccountLimit> accountLimits;
```
Comment (mbalc) [inline server/src/main/java/com/omigost/server/data/model/Account.java]: `List <User> users`

File: server/src/main/java/com/omigost/server/data/model/GroupLimit.java [hunk 3]
```diff
@@ -0,0 +1,13 @@
+package com.omigost.server.data.model;
```
Comment (mbalc) [inline server/src/main/java/com/omigost/server/data/model/GroupLimit.java]: We'll probably need some interfaces/mother classes or sth so we don't have to duplicate logic for GroupLimit and AccountLimit - there will probably be some
not to do in this PR but noticing

File: server/src/main/java/com/omigost/server/data/model/Account.java (hunk 2 above)
Comment (Ogochi) [reply server/src/main/java/com/omigost/server/data/model/Account.java]: We don't need currently. We can add it easily later.

**Comment by Ogochi** [comment]
@mbalc Due to decision to use AWS Budgets I changed models a bit.

**Comment by Ogochi** [comment]
WIP

**Comment by Ogochi** [comment]
scheme

File: server/src/main/java/com/omigost/server/data/model/GroupLimit.java (hunk 3 above)
Comment (styczynski) [reply server/src/main/java/com/omigost/server/data/model/GroupLimit.java]: @mbalc Good point

**Pull request closed without merging**